201-create-vm-vnet-nsg-avset-loadbal-rdpIn/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "201-create-vm-vnet-nsg-avset-loadbal-rdpIn",
```

Comment:
Better name would be something along the lines of "Setup NSG & NAT rules for RDP access to a VM"